source/CoordinateConversion/ProAppCoordConversionModule/ViewModels/TabBaseViewModel.cs

```diff
@@ -25,9 +25,10 @@
 using System.Windows.Controls;
 using System.Diagnostics;
 
-namespace CoordinateConversionLibrary.ViewModels
+
+namespace ProAppCoordConversionModule.ViewModels
 {
-    public class TabBaseViewModel : BaseViewModel
+    public class TabBaseViewModel : NotificationObject
```

Comment:
Instead of NotificationObject, I would use the Pro sdk provided ViewModelBase https://pro.arcgis.com/en/pro-app/sdk/api-reference/#topic10691.html